File: EPRO.Library.v3.5/EPRO.Library.v3.5/Program.cs
using CODERiT.Logger.v._3._5;

namespace ELibrary.Standard
{
    internal class Program
    {
        static Program()
        {
            ____Logger = new Log1(typeof(Program)).Load(Log1.Modes.FILE, true);
        }

        private static Log1 ____Logger;

        public static Log1 Logger
        {
            get
            {
                return ____Logger;
            }
        }
    }
}

File: EPRO.Library.v3.5/EPRO.Library.v3.5/Program.vb

Imports CODERiT.Logger.v._3._5




Friend Class Program


    Shared Sub New()
        ____Logger = New Log1(GetType(Program)).Load(Log1.Modes.FILE, True)

    End Sub




    Private Shared ____Logger As Log1
    Public Shared ReadOnly Property Logger As Log1
        Get
            Return ____Logger
        End Get
    End Property





End Class
